telegrinder/tools/formatting/html.py

```diff
@@ -29,14 +30,23 @@ def __post_init__(self) -> None:
 
 
 @dataclasses.dataclass(repr=False)
-class CodeBlock:
+class PreCode:
     string: str
     lang: str | None = None
```

Comment:
perhaps its better to use literal or enum

Reply:
Didn't understand... literal for 'lang' field?